controllers/databasecluster_controller.go

```diff
@@ -430,14 +449,23 @@ func (r *DatabaseClusterReconciler) genPSMDBBackupSpec(
 		switch backupStorage.Spec.Type {
 		case everestv1alpha1.BackupStorageTypeS3:
 			storages[schedule.BackupStorageName] = psmdbv1.BackupStorageSpec{
-				Type: psmdbv1.BackupStorageType(backupStorage.Spec.Type),
+				Type: psmdbv1.BackupStorageS3,
 				S3: psmdbv1.BackupStorageS3Spec{
 					Bucket:            backupStorage.Spec.Bucket,
 					CredentialsSecret: backupStorage.Spec.CredentialsSecretName,
 					Region:            backupStorage.Spec.Region,
 					EndpointURL:       backupStorage.Spec.EndpointURL,
 				},
 			}
+		case everestv1alpha1.BackupStorageTypeAzure:
+			storages[schedule.BackupStorageName] = psmdbv1.BackupStorageSpec{
+				Type: psmdbv1.BackupStorageAzure,
+				Azure: psmdbv1.BackupStorageAzureSpec{
+					Container:         backupStorage.Spec.Bucket,
+					Prefix:            azureStoragePrefix,
+					CredentialsSecret: backupStorage.Spec.CredentialsSecretName,
```

Comment:
Setting resources is required now for backup pods
https://github.com/percona/everest-operator/commit/2636ed4e5e4979bf26cd5b9cc97e86f84b2e042e

Reply:
I don't think I understand.
They're set in https://github.com/percona/everest-operator/blob/be76a10d78c5ba6c2dab90c5fc839d3835b9320b/controllers/databasecluster_controller.go#L303-L308

Are there any other resource limits to be configured?

Reply:
They are set by default for psmdb, but for PXC the configuration is set per storage
https://github.com/percona/everest-operator/blob/be76a10d78c5ba6c2dab90c5fc839d3835b9320b/controllers/databasecluster_controller.go#L938-L943

Reply:
as for PG they set by default as well and there's no need to add any changes

Reply:
ok, so for PG no action.
What would be your expectation for PXC?
We already set the resource limits in https://github.com/percona/everest-operator/blob/be76a10d78c5ba6c2dab90c5fc839d3835b9320b/controllers/databasecluster_controller.go#L938-L944

This applies to both S3 and Azure storages. Am I missing anything here?

Reply:
Yeah. That works. Thanks